### PR TITLE
ENH: CCG: enable to give forces as initial guess. usefull for making …

### PR DESCRIPTION
**Pull request by AntoineSIMTEK**
…restarts

### CODE REVIEW

File: ContactMechanics/Optimization/ConstrainedConjugateGradients.py [hunk 1]
```diff
@@ -41,11 +41,13 @@
 def constrained_conjugate_gradients(substrate, topography, hardness=None,
                                     external_force=None, offset=None,
                                     disp0=None,
```
Comment (pastewka) [inline ContactMechanics/Optimization/ConstrainedConjugateGradients.py:43]: Can we rename `disp0` to `initial_displacements` and `forces0` to `initial_forces`?

**Review by pastewka — CHANGES_REQUESTED** [review]
Looks good. I suggested a name change. Please also add `initial_forces` to the documentation string.

File: ContactMechanics/Optimization/ConstrainedConjugateGradients.py (hunk 1 above)
Comment (AntoineSIMTEK) [reply ContactMechanics/Optimization/ConstrainedConjugateGradients.py:43]: Agreed. This is the perfect moment to do that change since it is an API change.

Comment (pastewka) [reply ContactMechanics/Optimization/ConstrainedConjugateGradients.py:43]: Please also update the change log to reflect this API change

**Review by pastewka — DISMISSED** [review]

**Review by pastewka — APPROVED** [review]

**Pull request merged**